### ISSUE TITLE
question about loss function

### ISSUE THREAD

**Issue by cersar** [TEXTurePaper/TEXTurePaper#26]
Thanks for your greate work!
I have some confusion regarding the following loss function：
`loss = ((masked_pred - masked_target.detach()).pow(2) * masked_mask).mean() + (
                    (masked_pred - masked_pred.detach()).pow(2) * (1 - masked_mask)).mean()`
The second term calculates the squared difference between `masked_pred` and its detached version, which would always be 0. What is the purpose of including this term in the loss function?Thank you in advance.

**Comment by smha-Promedius** [comment]
I also agree with @cersar's opinion. 

I have also found that the second term of the loss function doesn't affect the final result. Ignoring that part, check the result in the following way;

```
self.Huberloss = torch.nn.HuberLoss().to(self.device)
loss = self.Huberloss(masked_pred, masked_target.detach())

...

loss += self.Huberloss(masked_current_z_normals, masked_last_z_normals.detach())
```

**Comment by eladrich** [comment]
Hi @cersar , this seems to be a leftover from one of our experiments. Thanks for noticing!
